### PR TITLE
Issue Fixed #7080

### LINKED ISSUE #7080: Add Option button should shift downward for adding a new option in Bundle Product.
### Terms

- [X] Before you openning this issue, i have checked if the issue has already been reported.

### Bagisto Version(s) affected

MASTER

### Issue Description

When we create a **bundle product,** there we can see **Add option Button**, Once we add Options and we need to create a new Option, there we need to **scroll up** and then we select the add option button. It would be **better** if the **Add Option button** **shifts downward.**

### Preconditions

**1.** Simple product should be created so that to add the product to bundle options.
**2.** At least one option should be added, so that we can add new option.

### Steps to reproduce

1. Go to Admin -> Create a Bundle Product -> Add Options.
2. After adding First option, now create another option.
3.  Here we need to scroll up and then we select the add option button.

### Expected Result

It would be much better if the Add Option Button shifts downward after we add a new option. So that if we need to create another option so we would not scroll up for the button, in bottom we can see the button we can add directly.

### Actual Result

Please check the **video** for a better **understanding** of the **enhancement** what we can do in it.

**Video**

https://user-images.githubusercontent.com/92721837/199908038-f1e70d31-9080-434d-b694-6a480c81426c.mp4

### Screenshots

Please check the screenshot where we can add the button.

Add option

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by JunaidJaved248**
## Issue Reference

#7080 
## Description

AddOption Button Moved Down
## How To Test This?

Go to Admin -> Create a Bundle Product -> Add Options.
After adding First option, now create another option.
Here we need to scroll up and then we select the add option button.
## Documentation
- [ ] My pull request requires an update on the documentation repository.

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
@JunaidJaved248 
Please add the **border** for the add [option button](https://prnt.sc/v4bFtYmjPmxI), please take a **reference** from [Here](https://prnt.sc/JBTZCN9OSQtO)

**Comment by JunaidJaved248** [comment]
@Sanjaybhattwebkul I think, there is no need of border here,
Border that You are referring, its accordian border,   And AddOption button is in under accordian,
In attached ss, you can see that when addoption button accordian closed, It seems Fine
image

**Comment by Sanjaybhattwebkul** [comment]
@JunaidJaved248 
Can you please add a **message** if there is **no product added** to a **bundle**, please take a **reference** from [Here](https://prnt.sc/LQIZpyGgSz2j), Actually there is **mismatching** between bundle products and the **add options** button?

**Comment by Sanjaybhattwebkul** [comment]
@JunaidJaved248 
This is **working fine**, Please find the **screenshot** [Here](https://prnt.sc/7DZNX1z60zVi)

**Comment by Sanjaybhattwebkul** [comment]
@JunaidJaved248 
**Thanks** for your **contribution**.

**Review by Sanjaybhattwebkul — APPROVED** [review]

File: packages/Webkul/Admin/src/Resources/views/catalog/products/accordians/bundle-items.blade.php [hunk 1]
```diff
@@ -378,5 +391,9 @@
                 }
             }
         });
+
+        Vue.component('no-product-added', {
```
Comment (jitendra-webkul) [inline packages/Webkul/Admin/src/Resources/views/catalog/products/accordians/bundle-items.blade.php:395]: I don't think there is a need for adding a new component

**Comment by devansh-webkul** [comment]
Closing the pull request, as I am not seeing any activity on this. Feel free to open a new one when the mentioned changes have been done.

**Pull request closed without merging**